Tests/pens/freetypePen_test.py

```diff
@@ -31,84 +30,92 @@ def star(pen):
     pen.lineTo((800, -200))
     pen.closePath()
 
+# Assume the buffers are equal when PSNR > 38dB.
```

Comment:
pardon my ignorance, I had to google what "PSNR" means. Maybe a little explanation in a comment or a link to the wikipedia page for https://en.wikipedia.org/wiki/Peak_signal-to-noise_ratio could help clarify

Reply:
also i'm curious about the 38db magic number, does it have any special meaning or is just some abitrary cut-off value?

Reply:
Thanks for the comment. It is a arbitrary cut-off value – at first I randomly took the `draw()` function from another test, and then I realised that I needed a test for quadratic curves as well, so `draw_quadratic()` contains a quadratic contour converted from the cubic one. That's where I chose `38db` as a threshold just to make the test pass when comparing the rendering results. I think the value is neither strict nor loose, but I don't have a good reasoning for the threshold. PSNR was one of the the easiest metrics for me to implement for comparison, but maybe there is better metrics for such a redering test.

Reply:
I see, thanks. Honestly I'm not so sure about embedding those base64-encoded zlib-compressed binary blobs for the expected renderings in the freetypePen_test.py.. There's no way a reader could make any sense of them without _looking at_ them as an image with pixels on screen. 
Maybe it's not fonttools's job to validate what the exact freetype rendering should be, I believe we could simply accept whatever comes out of freetype as good (provided it doesn't crash python).. What do you think?

Reply:
I agree that the tests are not for tracking the changes of FreeType. Still, I think they are valuable to verify the pen is emitting something meaningful. The reason I embedded test data in the source was because I was just hesitant to add test random images to the repo...and I've just come up with a simpler way to store them without size bloats in the latest commits. Please let me know if you have any concerns.